letsencrypt-auto-source/pieces/bootstrappers/deb_common.sh

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env bash
+#!/usr/bin/env bash
```

Comment:
It looks like you never pushed the change removing these `#!/usr/bin/env bash` lines. Could you push it?